tests/integration/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why do we need this?

the logging is already configured - https://github.com/apache/incubator-pulsar/blob/master/buildtools/src/main/resources/log4j2.xml

you can enable logging to console by specifying `-DredirectTestOutputToFile=false`

https://github.com/apache/incubator-pulsar/blob/master/pom.xml#L132